code/game/objects/items/plushes.dm

```diff
@@ -37,7 +37,9 @@
 /obj/item/toy/plush/random_snowflake/Initialize(mapload, set_snowflake_id)
 	. = ..()
 	var/list/configlist = CONFIG_GET(keyed_list/snowflake_plushies)
-	var/id = pick(configlist)
+	var/id = safepick(configlist)
```

Comment:
Somethins something while I like not having this runtime happen every single round, you should make sure the configs actually work if you haven't done so already and then ping headmins with all your might until it's fixed server side.